src/main/webapp/script.js

```diff
@@ -1,32 +1,28 @@
-function getRandomRestaurant(restaurants) {
-  // Pick a random restaurant.
-  const restaurant = restaurants[Math.floor(Math.random() * restaurants.length)];
-
-  // Add it to the page.
-  const resElement = document.getElementById('restaurant-name');
-  resElement.innerHTML = "<p>" + restaurant.name + "</p><p>" + JSON.stringify(restaurant.geometry.location) + "</p>";
-}
```

Comment:
We will need to implement this function in another way as this is the main purpose of the app

Reply:
Currently auto selects first restaurant in the list.